### PR TITLE
Convert md tutorials to rst and copy them to the website

### PR DESCRIPTION
**Pull request by gvegayon**
- [ ] Adds a recipe to `model/Makefile` to copy (and build if needed) the tutorials to the website in rst format.
- [ ] Removes non-human generated files (the `build` folder).
- [ ] Updates the index of the website to feature the tutorials.

### CODE REVIEW

**Review by dylanhmorris — COMMENTED** [review]
Looks good but needs manual conflict resolution @gvegayon

File: model/src/pyrenew/latent/hospitaladmissions.py [hunk 1]
```diff
@@ -13,23 +13,22 @@
 
 class HospAdmissionsSample(NamedTuple):
     """
-    A container for holding the output from HospAdmissionsSample.sample.
+        A container for holding the output from HospAdmissionsSample.sample.
+    >>>>>>> origin/main
```
Comment (dylanhmorris) [inline model/src/pyrenew/latent/hospitaladmissions.py:17]: Merge artifiact
```suggestion
```

**Review by dylanhmorris — APPROVED** [review]

**Comment by dylanhmorris** [comment]
@gvegayon looks as through there are some CI issues that need addressing

**Comment by gvegayon** [comment]
It was pre-commit and codecov is failing (server side). I'll try to re-run it.

**Pull request merged**